contrib/mesos/ci/test-conformance.sh

```diff
@@ -31,4 +31,14 @@ TEST_ARGS="$@"
 
 KUBE_ROOT=$(cd "$(dirname "${BASH_SOURCE}")/../../.." && pwd)
 
-"${KUBE_ROOT}/contrib/mesos/ci/run-with-cluster.sh" KUBECONFIG=~/.kube/config hack/conformance-test.sh ${TEST_ARGS}
+TEST_CMD=""
+if [ -n "${CONFORMANCE_BRANCH}" ]; then
+	CI_SHA1=$(git log -n 1 --format="%h" HEAD contrib/mesos/ci)
```

Comment:
This directory is hardcoded everywhere in the scripts. I don't want to clean this up here.